assets/css/woocommerce/woocommerce.scss

```diff
@@ -1871,7 +1871,7 @@ p.stars {
 	display: inline-block;
 	margin-bottom: 1em;
 	border-radius: 3px;
-	position: relative;
+	position: absolute;
```

Comment:
Instead of editing the style for `.onsale` class too shall we limit this to just:

```css
.wc-block-grid__product-onsale {
    position: absolute;
}

```